### PR TITLE
libvirt: update script to config libvirt on RHEL.

### PR DESCRIPTION
**Pull request by wyuany**
Update script to config libvirt not only work with Ubuntu but also RHEL 

Fixes: https://github.com/confidential-containers/cloud-api-adaptor/issues/1821

### CODE REVIEW

File: src/cloud-api-adaptor/libvirt/config_libvirt.sh [hunk 1]
```diff
@@ -82,8 +96,10 @@ echo "Installing Libvirt..."
 installLibvirt
 echo "Installing kcli..."
 installKcli
-echo "Installing kubectl..."
-installK8sclis
+if [ $OS_DISTRO != "rhel" ]; then
```
Comment (huoqifeng) [inline src/cloud-api-adaptor/libvirt/config_libvirt.sh:99]: Any reason don't install installK8sclis on rhel?

**Review by beraldoleal — DISMISSED** [review]
Hi @wyuany this is welcome, thanks! LGTM, except it shows 3 commits. 

Do you mind rebasing/squashing them?

**Review by stevenhorsman — APPROVED** [review]
+1 to Beraldo's suggestion to squash the commits, otherwise it LGTM

**Review by huoqifeng — APPROVED** [review]
LGTM

**Pull request merged**